apiserver/facades/client/applicationoffers/applicationoffers_test.go

```diff
@@ -768,32 +768,31 @@ func (s *applicationOffersSuite) TestShowErrorMsgMultipleURLs(c *gc.C) {
 	s.applicationOffers.listOffers = func(filters ...jujucrossmodel.ApplicationOfferFilter) ([]jujucrossmodel.ApplicationOffer, error) {
 		return nil, nil
 	}
-	s.mockState.model = &mockModel{uuid: s.mockState.modelUUID, name: "prod", owner: "fred@external", modelType: state.ModelTypeIAAS}
-	anotherModel := &mockModel{uuid: "uuid2", name: "test", owner: "fred@external", modelType: state.ModelTypeIAAS}
 
 	s.mockStatePool.st["uuid2"] = &mockState{
 		modelUUID: "uuid2",
```

Comment:
We can remove modelUUID from mockState also.
The suite has a mockState struct assigned in set up. The state uuid is set to `modeltesting.GenModelUUID(c).String()`. Just make modelUUID an attribute on the suite and use that instead of `state.modelUUID`

Reply:
Done & Done